install/kubernetes/helm/istio/test-values/values-istio-auth-sds.yaml

```diff
@@ -10,6 +10,8 @@ global:
     enabled: true
     udsPath: "unix:/var/run/sds/uds_path"
     useNormalJwt: true
+    useTrustworthyJwt: false
+    legacyJwtAllowed: true
```

Comment:
```useNormalJwt``` indicate use legacy jwt, why introduce another field ?

Reply:
Are you talking about `legacyJwtAllowed`? By default Citadel won't accept/allow legacy jwts, this flag is to override that behavior.

Reply:
yes; if useTrustworthyJwt==true, citadel use trustworthy jwt. otherwise fallback to normal jwt,
add a flag in citadel code sounds reasonable, in helm seems a bit redundant

Reply:
I pushed a new change to resolve this. PTAL.